src/lib/host/aws.js

```diff
@@ -123,6 +121,8 @@ module.exports = {
       spawn('shutdown', ['-h', 'now']);
     }
 
+    log('read userdata', { text: _.omit(userdata, ['securityToken']) });
+
```

Comment:
In theory this _host_ isn't used anymore.. is it?

Reply:
What do you mean?

Reply:
We use the `worker-runner` host for everything, so the aws host shouldn't matter (in fact, I probably should have deleted it).

Reply:
Ah, this is just part of the `git revert` command.

Reply:
Ah, OK.